CIP6/CIP6.md

```diff
@@ -0,0 +1,885 @@
+            "logo_url_png_logo": "https://mycoolpool.com/logo.png",
+            "color_main": "#RRGGBB",
+            "logo_vector": "<?xml version='1.0' ?><!DOCTYPE svg  PUBLIC '-//W3C//DTD SVG 1.1//EN'  'http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd'><svg enable-background='new 0 0 24 24' height='24px' id='Layer_1' version='1.1' viewBox='0 0 24 24' width='24px' xml:space='preserve' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'><g><path d='M12,0C5.4,0,0,5.4,0,12s5.4,12,12,12s12-5.4,12-12S18.6,0,12,0z M12,22C6.5,22,2,17.5,2,12S6.5,2,12,2s10,4.5,10,10   S17.5,22,12,22z'/><path d='M12,9c-1.1,0-2,0.9-2,2c0,0.6,0.3,1.1,0.7,1.5C10.3,12.9,10,13.4,10,14c0,1.1,0.9,2,2,2s2-0.9,2-2c0-0.6-0.3-1.1-0.7-1.5   c0.4-0.4,0.7-0.9,0.7-1.5C14,9.9,13.1,9,12,9z M12,15c-0.6,0-1-0.4-1-1s0.4-1,1-1s1,0.4,1,1S12.6,15,12,15z M12,12   c-0.6,0-1-0.4-1-1s0.4-1,1-1s1,0.4,1,1S12.6,12,12,12z'/><path d='M12,6c-3.3,0-6,2.7-6,6s2.7,6,6,6s6-2.7,6-6S15.3,6,12,6z M12,17c-2.8,0-5-2.2-5-5s2.2-5,5-5s5,2.2,5,5S14.8,17,12,17z'/></g></svg>"
+        },
+        "social": {
+            "twitter_handle": "coolpool",
+            "telegram_handle": "coolpool",
+            "facebook_handle": "coolpool",
+            "youtube_handle": "coolpool",
+            "twitch_handle": "coolpool",
+            "discord_handle": "coolpool",
+            "github_handle": "coolpool"
+        },
+        "contact": {
+            "abuse": "abuse@pooldomain.org",
+            "support": "help@pooldomain.org"
+        },
+        "company": {
+            "name": "Company Name",
+            "addr": "Street, Number",
+            "city": "London",
+            "country": "UK",
+            "company_id": "123456789",
+            "vat_id": "GB123456789"
+        },
+        "operator": {
+            "country": "UK",
+            "sex": "FEMALE"
+        },
+        "about": {
+            "team_affiliation": [
+                "ISPPA",
+                "Cardano Ambassador"
+            ],
+            "me": "Long description of me",
+            "server": "long description of server details",
+            "company": "long description of company details"
+        },
+        "rss": "https://mycoolpool.com/xml/poolrss.xml"
+    },
+    "pools": [{
+            "id": "0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f",
+            "country": "UK",
+            "os": "LINUX",
+            "infrastructure": "AWS",
+            "status": "act",
+            "saturated_recommend":"0a0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f"
+        },
+        {
+            "id": "0a0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f",
+            "country": "UK",
+            "os": "LINUX",
+            "infrastructure": "AWS",
+            "status": "act",
+            "saturated_recommend":"0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f"
+        }
+    ]
+}
+```
+
+#### Extended Metadata Schema
+
+```
+{
```

Comment:
This would be better in a separate file as a `json-schema.json` so that it can be viewed / fetched independently.

Reply:
Ok, will move it in the next iteration of the PR